openedx/core/djangoapps/programs/migrations/0007_auto_20160329_1031.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
You've got two migrations in here when you only need one. Delete them both, then re-create a fresh migration to reflect the most recent state of your models.